src/cli/add.rs

```diff
@@ -73,6 +83,29 @@ pub struct Args {
     /// Whether the pypi requirement should be editable
     #[arg(long, requires = "pypi")]
     pub editable: bool,
+
+    /// Where to add the pypi requirement to the manifest
+    #[arg(long, requires = "pypi")]
+    pub location: Option<PypiDependencyLocation>,
+}
+
+#[derive(Parser, Debug, Clone, PartialEq, ValueEnum)]
+pub enum PypiDependencyLocation {
```

Comment:
What about adding the `dependencies-groups` here as well so you are allowed to be explicit?

Reply:
Sure